parser_library/test/debugging/debugger_test.cpp

```diff
@@ -347,14 +347,7 @@ TEST(debugger, test)
             },
             std::unordered_map<std::string, test_var_value> {
                 // macro locals
-                {
-                    "&SYSLIST",
-                    test_var_value("(10,13)",
-                        list {
-                            { "0", std::make_shared<test_var_value>("10") },
-                            { "1", std::make_shared<test_var_value>("13") },
-                        }),
-                },
```

Comment:
When the `SYSLIST` is unsubscripted it will default to `SYSLIST(1)` hence the need for change in this test.
See the following for more details: https://www.ibm.com/docs/en/zos/2.1.0?topic=messages-asma096e

If the aim of this test is to verify that there are only two subscript-value pairs in SYSLIST I can try to modify this test to do so.

Reply:
This is testing the macro tracer component, so I think that the expected content in the VSCode debugger is (10,13) next to the SYSLIST variable, with the component break-down of 0:10 and 1:13.